tests/UnitTests/utcommon/inc/utcommon.h

```diff
@@ -29,6 +29,8 @@ void k4a_unittest_init_logging_with_processid();
 #endif
 
 int k4a_test_common_main(int argc, char **argv);
+
+int64_t k4a_unittest_get_max_sync_delay(k4a_fps_t fps);
```

Comment:
nit: Can you add either a comment of change this function name to include units of `max_delay`?

```suggestion
int64_t k4a_unittest_get_max_sync_delay_ms(k4a_fps_t fps);
```